cmd/argo/commands/archive/list.go

```diff
@@ -25,6 +25,18 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list workflows in the archive",
+		Example: `# List all Archived Workflows:
+  argo archive list
+  
+# List Archived Workflows in YAML format:
+  argo archive list --output yaml
+  
+# List Archived Workflows with a specific chunk size:
+  argo archive list --chunk-size <chunk_size>
+  
+# List Archived Workflows with the specified labels (used to filter archived workflows):
+  argo archive list -l key1=value1,key2=value2
```

Comment:
```suggestion
# List Archived Workflows that have both labels:
  argo archive list -l label1=value1,label2=value2
```

Let's be consistent with https://github.com/argoproj/argo-workflows/pull/12076#discussion_r1372149059